packages/esm-patient-registration-app/src/patient-registration/field/field.test.tsx

```diff
@@ -288,9 +288,7 @@ describe('Field', () => {
       error = err;
     }
 
-    expect(error).toBe(
-      "Invalid field name 'invalidField'. Valid options are 'weight', 'name', 'gender', 'dob', 'address', 'id', 'phone & email'.",
-    );
+    expect(error).toMatch(/Invalid field name 'invalidField'. Valid options are /);
```

Comment:
The error message is incomplete.

Reply:
It's intentional. With the list, then every time the built-ins change the test needs updating. Without, it fails to test against the possibility that something goes wrong with listing the built-ins. It's a minor preference, but I think the risk of the latter is worth the convenience of avoiding the former.